### PR TITLE
Remove the prefix from fake location case sharing groups

### PR DESCRIPTION
**Pull request by twymer**
This is an update to match Sheel's request

### CODE REVIEW

File: corehq/apps/fixtures/models.py [hunk 1]
```diff
@@ -289,6 +289,38 @@ def _serialize(val):
...
+            groups.append(reporting_location.reporting_group_object())
+
+        return groups
+
+    def _get_case_sharing_groups(self, group_ids):
+        groups = []
+
+        case_sharing_locations = set(SQLLocation.objects.filter(
+            location_id__in=group_ids
+        ).values_list('location_id', flat=True))
```
Comment (czue) [inline corehq/apps/fixtures/models.py]: not sure if it makes a difference performance wise, but you can use `.distinct()` to push the uniqueness to the database.

**Comment by czue** [comment]
looks good to me, kicked build. does this need QA? are we just rolling out breaking changes to people using the old scheme?

**Comment by sheelio** [comment]
I think we're going to QA it once more now that this is done.  Any existing projects will be broken until they sync (tho I didn't see any real projects using it).

File: corehq/apps/fixtures/models.py (hunk 1 above)
Comment (twymer) [inline corehq/apps/fixtures/models.py]: I'm sure packing a list into a set (then pulling it back out) is slower than relying on SQL, so fixed that.

**Comment by twymer** [comment]
@czue @esoergel We decided to ship it and QA on prod, since this feature is not actually "live" yet.

**Comment by czue** [comment]
@twymer even though the build is passing, I think if we merge this tests are going to fail, as per the comment in #6965 can you update?

**Comment by czue** [comment]
(no blockers on merge from me besides that. am generally a little skeptical this could cause problems so good to make sure the support team/nate is ready for this. can you make sure @amsagoff and @sheelio sign off on a communication path?)

**Comment by twymer** [comment]
@czue oooh, will do

**Comment by twymer** [comment]
@czue updated, pending build

**Comment by sheelio** [comment]
@czue I reviewed all the apps on prod using this functionality yesterday.  They're all test projects and I'm communicating to each of the project owners.  Will also flag this to Nate as soon as it gets on prod.

**Comment by czue** [comment]
@esoergel @orangejenny i'm going to defer to you for final merge/deploy coordination in case there is any. nothing blocking from me

**Comment by orangejenny** [comment]
@esoergel let me know if there's anything special deploy-wise for this

**Comment by czue** [comment]
whee

**Pull request merged**